pkgs/development/tools/misc/pkgconf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, automake, autoconf, libtool }:
+
+stdenv.mkDerivation rec {
+  name = "pkgconf-1.5.4";
+
+  src = fetchgit {
+    url = "https://git.dereferenced.org/pkgconf/pkgconf.git";
+    rev = "74133eda31bc1ed5947b4a3a854001e320b6c1fe";
+    sha256 = "159fxbwm5shz8p95jp28wrjvinlhmp42dy60pqg34psjn41wq1q4";
+  };
+
+  buildInputs = [ automake autoconf libtool ];
+
+  preConfigurePhases = ["autogenPhase"];
+
+  autogenPhase = ''
+    ./autogen.sh
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Package compiler and linker metadata toolkit";
+    homepage = https://git.dereferenced.org/pkgconf/pkgconf;
```

Comment:
https://pkgconf.org/

Reply:
The website is clearly out-of-date and returns an invalid HTTPS certificate. I would recommend against linking it in the derivation. It also points to GitHub, which is not the official repository any longer.

Reply:
Any more concerns on this? The rest should be fixed.